lib/python/util.py

```diff
@@ -34,3 +34,21 @@ def list_files_recursively(dir, pattern):
             matches.append(os.path.join(root, filename))
 
     return matches
+
+def get_git_revision_script_url(file_path):
+    import os
+    import subprocess
```

Comment:
According to [PEP 8](https://www.python.org/dev/peps/pep-0008/#id23) all imports should be at the top of a file.  I guess there may be cases when you only want to import into a function's local namespace, but I don't think that is the case here.